configure

```diff
@@ -2806,6 +2806,7 @@ case $_host_os in
 		append_var CXXFLAGS "-mcpu=750"
 		append_var CXXFLAGS "-meabi"
 		append_var CXXFLAGS "-mhard-float"
+		append_var CXXFLAGS "-fpermissive"
```

Comment:
That's unfortunate, is there a libogc define you can use to differentiate the prototype?

Reply:
Hum, in fact I didn't notice that the change of prototype is quite recent and thought a change of compiler transformed a warning into error.
I see now that libogc has changed prototype between 1.8.17 and 1.8.18, I will try to use a #ifdef to avoid -fpermissive

Reply:
I did the fix with #ifdef.